app/models/dhs1171_pdf.rb

```diff
@@ -3,23 +3,29 @@ class Dhs1171Pdf
   SOURCE_PDF = "#{PDF_DIRECTORY}/DHS_1171.pdf".freeze
   COVERSHEET_PDF = "#{PDF_DIRECTORY}/michigan_snap_fax_cover_letter.pdf".freeze
 
-  def initialize(snap_application:, output_filename: "test_output.pdf")
+  def initialize(snap_application:)
     @snap_application = snap_application
-    @completed_filename = "tmp/completed#{output_filename}"
-    @output_filename = "tmp/#{output_filename}"
   end
 
-  def save
-    fill_in_template_form
-    add_cover_sheet_to_completed_form
+  def completed_file
+    complete_template_pdf_with_client_data
+    prepend_cover_sheet_to_completed_form
+    complete_form_with_cover
+  ensure
```

Comment:
If the `tempfile` is not explicitly closed, it will hang around until the next Garbage Collection. Not a huge deal but best practice is to clean up after yourself.

`ensure` is a block modifier which ensures whatever is inside will get run regardless of any exceptions or interruptions.